scipy/special/_loggamma.h

```diff
@@ -0,0 +1,177 @@
+#pragma once
+
+/* An implementation of the principal branch of the logarithm of
+ * Gamma. Also contains implementations of Gamma and 1/Gamma which are
+ * easily computed from log-Gamma.
+ *
+ * Author: Josh Wilson
+ * Translated from Cython to C++ in 2023 by SciPy developers.
+ *
+ * Distributed under the same license as Scipy.
+ *
+ * References
+ * ----------
+ * [1] Hare, "Computing the Principal Branch of log-Gamma",
+ *     Journal of Algorithms, 1997.
+ *
+ * [2] Julia,
+ *     https://github.com/JuliaLang/julia/blob/master/base/special/gamma.jl
+ */
+
+#include <cmath>
+#include <complex>
+
+#include "cephes.hh"
+#include "special/evalpoly.h"
+#include "_zlog1.h"
+#include "_trig.h"
+#include "special/error.h"
+
+
+namespace special {
+    using namespace std::complex_literals;
```

Comment:
I think best practice for this is to do `using namespace std::complex_literals;` at the top of the functions themselves, wherever I is used. It should work.